lib/vedeu/models/all.rb

```diff
@@ -1,15 +1,12 @@
 require 'vedeu/repositories/all'
 
-require 'vedeu/models/cell'
-require 'vedeu/models/char'
-require 'vedeu/models/stream'
-require 'vedeu/models/line'
-require 'vedeu/models/interface'
-require 'vedeu/models/composition'
+require_relative 'cell'
+require_relative 'char'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.